bench.csv

```diff
@@ -1,630 +1,482 @@
-"Agora/Proposal/validator/unlocking/illegal/with 42 proposals/retract votes while not voting/role: Both , status: Draft/stake",1893561675,4783196,29107
-"Agora/Proposal/validator/unlocking/illegal/with 42 proposals/retract votes while not voting/role: Both , status: Locked/stake",1893561675,4783196,29107
-"Agora/Proposal/validator/unlocking/illegal/with 42 proposals/retract votes while not voting/role: Both , status: Finished/stake",1893561675,4783196,29107
-"Agora/Proposal/validator/unlocking/illegal/with 42 proposals/retract votes while not voting/role: Irrelevant , status: Draft/stake",1346731285,3355886,27707
-"Agora/Proposal/validator/unlocking/illegal/with 42 proposals/retract votes while not voting/role: Irrelevant , status: Locked/stake",1346731285,3355886,27707
-"Agora/Proposal/validator/unlocking/illegal/with 42 proposals/retract votes while not voting/role: Irrelevant , status: Finished/stake",1346731285,3355886,27707
-Agora/Proposal/validator/unlocking/illegal/with 42 proposals/unlock an irrelevant stake/status: Draft retract votes: True/stake,1346731285,3355886,27707
-Agora/Proposal/validator/unlocking/illegal/with 42 proposals/unlock an irrelevant stake/status: Draft retract votes: False/stake,1346731285,3355886,27707
-Agora/Proposal/validator/unlocking/illegal/with 42 proposals/unlock an irrelevant stake/status: VotingReady retract votes: True/stake,1346731285,3355886,27707
-Agora/Proposal/validator/unlocking/illegal/with 42 proposals/unlock an irrelevant stake/status: VotingReady retract votes: False/stake,1346731285,3355886,27707
-Agora/Proposal/validator/unlocking/illegal/with 42 proposals/unlock an irrelevant stake/status: Locked retract votes: True/stake,1346731285,3355886,27707
-Agora/Proposal/validator/unlocking/illegal/with 42 proposals/unlock an irrelevant stake/status: Locked retract votes: False/stake,1346731285,3355886,27707
-Agora/Proposal/validator/unlocking/illegal/with 42 proposals/unlock an irrelevant stake/status: Finished retract votes: True/stake,1346731285,3355886,27707
-Agora/Proposal/validator/unlocking/illegal/with 42 proposals/unlock an irrelevant stake/status: Finished retract votes: False/stake,1346731285,3355886,27707
-Agora/Proposal/validator/unlocking/illegal/with 42 proposals/remove creator too early/status: Draft/stake,1583286885,3905144,28335
-Agora/Proposal/validator/unlocking/illegal/with 42 proposals/remove creator too early/status: VotingReady/stake,1583286885,3905144,28335
-Agora/Proposal/validator/unlocking/illegal/with 42 proposals/remove creator too early/status: Locked/stake,1583286885,3905144,28335
-Agora/Proposal/validator/unlocking/illegal/with 42 proposals/creator: retract votes/stake,1583286885,3905144,28251
+Agora/Effects/Treasury Withdrawal Effect/effect/Simple,657490362,1527566,8129
+Agora/Effects/Treasury Withdrawal Effect/effect/Simple with multiple treasuries ,944564452,2157214,8561
```

Comment:
Wow, it seems that things became a lot more expensive. Do you know why?

Reply:
I have no idea. It happened in `ddb428332f3e9ed7f0b28b19ecede2c58cc38745` but I didn't even touch the effect script in that commit.

Reply:
Oh wait, I think I know why:
<img width="1064" alt="image" src="https://user-images.githubusercontent.com/40087668/188897776-6228a2fa-00ad-4095-8c6d-0891acc87a73.png">

Reply:
That checks out. We probably want to reverse this?

Reply:
Yep, working on it.

Reply:
Done!